src/coreComponents/mesh/generators/PrismUtilities.hpp

```diff
@@ -42,7 +42,8 @@ localIndex getFaceNodesPrism( localIndex const faceNum,
 
   if( faceNum == 0 )
   {
-    GEOS_ERROR_IF_LT( faceNodes.size(), 4 );
+    GEOS_ERROR_IF_LT_MSG( faceNodes.size(), 4,
+                          "Invalid number of nodes for face (face index = " << faceNum << ")." );
```

Comment:
Can you factorize the error message?

Reply:
Are you suggesting this?
```c++
  static constexpr auto invalidNodeCountError = "Invalid number of nodes for face (face index = {})."

  if( faceNum == 0 )
  {
    GEOS_ERROR_IF_LT_MSG( faceNodes.size(), 4,
                          GEOS_FMT( invalidNodeCountError, faceNum ) );
    faceNodes[0] = elemNodes[0];
    faceNodes[1] = elemNodes[1];
    faceNodes[2] = elemNodes[N+1];
    faceNodes[3] = elemNodes[N];
    return 4;
  }
  [...]
  else if( faceNum == N + 1 )
  {
    GEOS_ERROR_IF_LT_MSG( faceNodes.size(), N,
                          GEOS_FMT( invalidNodeCountError, faceNum ) );
    for( localIndex i = 0; i <  N; ++i )
    {
      faceNodes[i] = elemNodes[i+N];
    }
    return N;
  }
  else
  {
    GEOS_ERROR( GEOS_FMT( "Invalid local face index for element of type Prism{}: {}", N, faceNum ) );
    return 0;
  }

```

Reply:
I made another proposal in the last commit: storing the messages in `string_views` and reusing them as you suggested.